src/main/java/WayofTime/bloodmagic/ritual/IMasterRitualStone.java

```diff
@@ -61,4 +65,16 @@
     default SoulTicket ticket(int amount) {
         return SoulTicket.block(getWorldObj(), getBlockPos(), amount);
     }
+
+    /**
+     * Used to grab the range of a ritual for a given effect.
+     *
+     * @param range - Range that needs to be pulled.
+     * @return -
+     */
+    AreaDescriptor getBlockRange(String range);
```

Comment:
Spacing between these 3 declarations and there's extra lines at the end.

Reply:
Done.